src/tests/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ set(sources
         MainOpt.cxx
         remote.cxx
         helper.cxx
-        uri.cxx
+        uri_tests.cxx
```

Comment:
What's the reason for renaming?

Reply:
The test files mirror the `src` filenames, and when refactoring sometimes the editor gets confused. Also when debugging tests it becomes difficult to differentiate between the two. The `_tests.cxx` is also more descriptive as it implies there are tests in the file rather than implementation.

Reply:
Alright, I guess it depends on the IDE.